contrib/completion/bash/docker

```diff
@@ -138,6 +138,10 @@ __docker_containers_and_images() {
 	COMPREPLY+=( "${containers[@]}" )
 }
 
+__docker_networks() {
+	COMPREPLY=( $(compgen -W "$(__docker_q network ls | awk 'NR>1 {print $2}')" -- "$cur") )
```

Comment:
Yes, sure. I was in doubt whether IDs are really useful and opted against them. How about making their inclusion configurable with the env var used for containers?